docker/main_test.go

```diff
@@ -14,7 +96,86 @@ func TestVolumeList(t *testing.T) {
     if err != nil {
         t.Fatal(err)
     }
-    if len(vols.Results) != 2 {
+    if len(vols.Results) == 0 {
         t.Fatal("No volumes found")
     }
 }
+
+func TestVolumeListByHost(t *testing.T) {
+    netbox.NETBOX_URL = os.Getenv("NETBOX_URL")
+    netbox.NETBOX_TOKEN = os.Getenv("NETBOX_TOKEN")
+   
+    host, err := HostSearchByName("testhost")
+    vols, err := VolumeListByHost(host.Id)
+    if err != nil {
+        t.Fatal(err)
+    }
+    if len(vols.Results) > 0 {
+        if vols.Results[0].Host.Name != "testhost" {
+            t.Fatal(vols.Results[0].Host.Name)
+        }
+    } else {
+        t.Fatal("No volumes found")
+    }
+}
+
+func TestContainerDelete(t *testing.T) {
+    netbox.NETBOX_URL = os.Getenv("NETBOX_URL")
+    netbox.NETBOX_TOKEN = os.Getenv("NETBOX_TOKEN")
+
+    host, err := HostSearchByName("testhost")
+    container, err := ContainerSearchByName(host, "testcontainer")
+    err = ContainerDelete(container.Id)
+    if err != nil {
+        t.Fatal(err)
+    }
+    
+}
+
+func TestVolumeDelete(t *testing.T) {
+    netbox.NETBOX_URL = os.Getenv("NETBOX_URL")
+    netbox.NETBOX_TOKEN = os.Getenv("NETBOX_TOKEN")
+
+    host, err := HostSearchByName("testhost")
+    volume, err := VolumeSearchByName("testvolume", host.Id)
+    err = VolumeDelete(volume.Id)
```

Comment:
function err(err) {  
if err != nil {
        t.Fatal(err)
    }

err(VolumeSearchByName("testvolume", host.Id))

Reply:
this is not idiomatic go, this is:

```go
result, err := FunctionCall(params)
if err != nil {
  t.Fatalf("unexpected error during <description>: %v", err)
}
```